### PR TITLE
Option Display SteamDb Rating

### PR DESCRIPTION
**Pull request by GayKevin**
- Add option to display SteamDb Rating on app pages

### CODE REVIEW

File: scripts/store/app.js [hunk 1]
```diff
@@ -283,51 +283,66 @@ else
...
+			element.id = 'steamdb_bind_tooltip';
+			element.type = 'text/javascript';
+			element.appendChild( document.createTextNode( 'BindStoreTooltip( jQuery( ".btn_steamdb > span" ) );' ) );
+
+			document.head.appendChild( element );
 		}
-	}
+
+		if( items[ 'link-steamdb-rating' ] )
+        {
```
Comment (xPaw) [inline scripts/store/app.js]: Keep tabs please.

File: scripts/store/app.js [hunk 2]
```diff
@@ -283,51 +283,66 @@ else
...
-		summary.appendChild( link );
-		container.appendChild( subtitle );
-		container.appendChild( summary );
-		
-		let element = document.querySelectorAll( '.user_reviews_summary_row' );
-		element = element[ element.length - 1 ];
-		
-		if( element )
+
+		// We need to bind a tooltip, and the only way to do that is to inject a script into the page
```
Comment (xPaw) [inline scripts/store/app.js]: Duplicated code.

File: options/options.html [hunk 3]
```diff
@@ -64,6 +64,11 @@
 			<span class="control-indicator"></span>
 			Add a calculator link on community profiles
 		</label>
+		<label class="control checkbox">
+			<input type="checkbox" class="option-check" checked data-option="link-steamdb-rating">
```
Comment (xPaw) [inline options/options.html]: Remove `link-` prefix.

Comment (GayKevin) [reply options/options.html]: - Changed to `steamdb-rating`

File: scripts/store/app.js (hunk 2 above)
Comment (GayKevin) [reply scripts/store/app.js]: Sorry, i deleted the duplicate code.

File: scripts/store/app.js (hunk 1 above)
Comment (GayKevin) [reply scripts/store/app.js]: It should be fix now

**Comment by xPaw** [comment]
It's still using spaces, not tabs.

File: scripts/store/app.js (hunk 1 above)
Comment (GayKevin) [reply scripts/store/app.js]: Convert all spaces to tabs

**Comment by xPaw** [comment]
Thanks!

**Pull request merged**